src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -349,17 +357,19 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(config : HostConfig<T, P,
 
   function commitAllLifeCycles() {
     while (nextEffect !== null) {
-      const current = nextEffect.alternate;
+      const effectTag = nextEffect.effectTag;
+
       // Use Task priority for lifecycle updates
-      if (nextEffect.effectTag & (Update | Callback)) {
+      if (effectTag & (Update | Callback)) {
```

Comment:
Outside the scope of this PR, but we should probably have a separate check for callbacks, for similar reasons.